src/proj_api.h

```diff
@@ -49,20 +49,21 @@ extern "C" {
  *   are padded with leading zeros if necessary); e.g., PJ_VERSION=401000
  *   for version 4.10.0.
  */
+#ifndef PROJ_H
```

Comment:
It seems that there's a provision to do #include "proj.h" and then #include "proj_api.h", but the reverse wouldn't work well, right ?